tests/api_test.py

```diff
@@ -24,25 +22,24 @@ def test_basepath_rewriting():
 def build_config(schema_dir):
     return mock.Mock(
         registry=get_registry({
-            'swagger_schema': compile_swagger_schema(schema_dir),
+            'swagger_schema': validate_swagger_schema(schema_dir),
         })
```

Comment:
Oh correct. I was testing my validation spec here which was wrong. I have reverted back these changes, and added similar tests in `spec_test.py` which get validated through `swagger_spec_validator`.